src/wram.asm

```diff
@@ -769,4 +785,4 @@ wMusicE4:: ; ddea
 	ds $9
 
 wMusicReturnAddress:: ; ddf3
-	ds $8
+	ds $8
```

Comment:
Try to watch out for creating unnecessary white space, and don't delete EOF newlines.